test/performance/odp_stress.c

```diff
@@ -73,8 +73,8 @@ static void print_usage(void)
 	       "Stress test options:\n"
 	       "\n"
 	       "  -c, --num_cpu          Number of CPUs (worker threads). 0: all available CPUs. Default: 1\n"
-	       "  -p, --period_ns        Timeout period in nsec. Default: 1 sec\n"
-	       "  -r, --rounds           Number of timeout rounds. Default: 10\n"
+	       "  -p, --period_ns        Timeout period in nsec. Default: 100 ms\n"
+	       "  -r, --rounds           Number of timeout rounds. Default: 1\n"
```

Comment:
This still could be more than one so whatever depends on this gets actually looped, which might reveal some issues.